.github/workflows/main.yml

```diff
@@ -116,7 +110,32 @@ jobs:
             cd ..
             coverage run -m pytest tests/forte/notebooks
           fi
-
+  test_backbone:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [ 3.6 ]
+        torch-version: [ 1.5.0 ]
+        tensorflow-version: [ 1.15.0 ]
```

Comment:
Maybe we don't need to put `pytorch` and `tensorflow` here in the matrix since they are not required in this job.